components/financial_assistance/app/domain/financial_assistance/operations/applications/aptc_csr_credit_eligibilities/renewals/request_all.rb

```diff
@@ -26,7 +26,8 @@ def call(params)
 
             def find_families(renewal_year)
               family_ids = ::HbxEnrollment.individual_market.enrolled.current_year.distinct(:family_id)
-              determined_family_ids = ::FinancialAssistance::Application.by_year(renewal_year.pred).where(:family_id.in => family_ids).distinct(:family_id)
+              # selection logic here matches the selection logic in ::FinancialAssistance::Operations::Applications::AptcCsrCreditEligibilities::Renewals::Renew
+              determined_family_ids = ::FinancialAssistance::Application.by_year(renewal_year.pred).determined.where(:family_id.in => family_ids).select(&:eligible_for_renewal?).map(&:family_id).uniq
 
```

Comment:
I think this may be a more accurate selection of renewal eligible applications
```ruby
 # All family IDs where the latest determined application is renewal eligible
renewal_eligible_family_ids = ::FinancialAssistance::Application.by_year(renewal_year.pred).determined.where(:family_id.in => family_ids).group_by(&:family_id).select { |_family_id, group| group.max_by(&:created_at).eligible_for_renewal? }.keys

Success(renewal_eligible_family_ids)
```